mailer.rst

```diff
@@ -129,6 +129,13 @@ Service               Install with
     <mail-catcher>` instead. Note that most supported providers also offer a
     free tier.
 
+.. note::
```

Comment:
shouldn't this go with the other notes below the table of schemes ? there, it could explain that all the `https` and `api` transports require the HttpClient.

Reply:
Good idea!